src/main/java/hebron/app/paypal/PayPallController.java

```diff
@@ -0,0 +1,43 @@
+package hebron.app.paypal;
+
+import com.paypal.api.payments.Amount;
+import com.paypal.api.payments.Payment;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class PayPallController {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(PayPallController.class);
+
+    private PayPalService payPalService;
+
+    @Autowired
+    public PayPallController(PayPalService payPalService) {
+        this.payPalService = payPalService;
+    }
+
+
+    @PostMapping("/api//paypal/createPayment")
+    public ResponseEntity createPayment(@RequestBody String totalAmount) {
+        Payment payment = payPalService.createPayment(totalAmount);
+        return ResponseEntity.ok(payment);
+    }
+
+    @PostMapping("/api/paypal/execute")
+    public ResponseEntity executePayment(@RequestBody PaymentDto paymentDto) {
+        LOGGER.debug("Execute Payment:[" + paymentDto + "]");
+        Payment executePayment = payPalService.executePaymentAfterApprove(paymentDto);
+        if (executePayment.getState().equalsIgnoreCase("approved")) {
```

Comment:
it is better to use:
private static final String APPROVED = "approved";
APPROVED.equalsIgnoreCase(executePayment.getState()) 
to avoid NullPointerException